synthetics/test_schema.go

```diff
@@ -102,61 +105,65 @@ func makeTestSettingsSchema(mode schemaMode) *schema.Schema {
 		"trace":               makeTestTraceSchema(mode),
 		"port": {
 			Type:     schema.TypeInt,
-			Optional: optionalOnCreate(mode),
+			Optional: optionalOnRead(mode),
+			Required: requiredOnCreate(mode),
 			Computed: computedOnRead(mode),
 		},
 		"protocol": {
 			Type:     schema.TypeString,
-			Optional: optionalOnCreate(mode),
+			Optional: true,
```

Comment:
AFAIK, protocol is required on create (and ignored for some test types :-/).

Reply:
Addressed: a2e73bcd0db6697b5392fdb69a572f74228bc594

Reply:
Done